boards/arm/mimx8mm_evk_a53/mimx8mm_evk_a53_defconfig

```diff
@@ -0,0 +1,29 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Platform Configuration
+CONFIG_SOC_SERIES_IMX8MM_A53=y
+CONFIG_SOC_MIMX8MM_A53=y
+CONFIG_BOARD_MIMX8MM_EVK_A53=y
+CONFIG_ARM64_VA_BITS_36=y
+CONFIG_ARM64_PA_BITS_36=y
+CONFIG_ARM_ARCH_TIMER=y
+
+# Zephyr Kernel Configuration
+CONFIG_SRAM_BASE_ADDRESS=0x40000000
+CONFIG_XIP=n
+CONFIG_SRAM_SIZE=1024
+CONFIG_ARM64_PA_BITS_48=y
+
+# Serial Drivers
+CONFIG_SERIAL=y
```

Comment:
you have `CONFIG_SERIAL=y` but no serial driver enabled?

Reply:
I have this in Kconfig

`
if SERIAL

config UART_MCUX_IUART
        default y

endif # SERIAL
`